esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2514,8 +2510,15 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
         MemoryBuffer& dstxml = (MemoryBuffer&)req.getDstxml();
         if(dstxml.length() == 0)
         {
-            if(isEmptyString(destPlane) && destip.isEmpty())
+#ifdef _CONTAINERIZED
+            if (isEmptyString(destPlane))
```

Comment:
I'm not sure that this should be conditional on containerized.  A different issue.